src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,135 @@
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+    <script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>
+
+    <style amp-custom>
+    .full-width {
+      width: 100%;
+    }
+    form.amp-form-submit-success [submit-success] {
+        width: 100%;
+        color: black;
+    }
+    form.amp-form-submit-success .data-input,
+    form.amp-form-submit-success .img-data-input, 
+    form.amp-form-submit-success .button{
+        display: none;
+    }
+    form.amp-form-submit-error [submit-error] {
+        color: red;
+    }
+    .date {
+      color: #a8a3ae;
+      font-size: 12px;
+    }
+    .comment {
+        background: #fff;
+        margin: 0 0 16px 0;
+        width:100%;
+    }
+    .nomargin {
+        margin: 0;
+    }
+    .space-left {
+      margin-left: 16px;
+    }
+    .nopadding {
+      padding: 0;
+    }
+    .comment-user {
+      display: flex;
+    }
+    </style>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script id="amp-access" type="application/json">
+    {
+        "authorization": [[.AuthorizationEndpoint]],
+        "pingback": "/samples_templates/comment_section/pingback?rid=READER_ID&ref=DOCUMENT_REFERRER&url=CANONICAL_URL",
+        "login": [[.LoginEndpoint]],
+        "authorizationFallbackResponse": {
+            "error": true,
+            "access": false
+        }
+    }
+    </script>
+    </head>
+    <body>
+    <!-- #### Add a Comment Section -->
+  <!--
+    Use the `amp-access` to realize a login flow. Learn more about amp-access [here](/components/amp-access). 
+    Use the `amp-form` component to realize a comment section. Learn more about amp-fomr [here](/components/amp-form).
+  -->
+  <div>
+    <span amp-access="user = 'test-user'" amp-access-hide>
+          <a href="<%host%>/samples_templates/[[.Path]]/logout" class="button button-primary space-left">Logout</a>
```

Comment:
amp-access trick: define a [separate login endpoint](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md#login-page) for the logout. This way you can use variable substitutions for the logout as well and provide a return URL. This means you don't need to separate between preview and sample page in the backend.